tests/visual/components/progress/index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Progress Component Tests · Stencil</title>
+    <link rel="stylesheet" href="../../../fixtures/fixture.css">
+
+    <link rel="stylesheet" href="progress.css">
+    <style>
+    </style>
+</head>
+<body>
+    <h1 class="c-fixture-title">
+        <a class="c-fixture-title__sub" href="https://github.com/mobify/stencil">Stencil</a>
+        <div>Progress Component Tests</div>
+    </h1>
+
+    <article class="c-test">
+        <h2 class="c-test__describe">.c-progress</h2>
+
+        <div class="c-test__case">
+            <p class="c-test__should">It should render a progress indicator.</p>
+            <div class="c-test__run">
+
+                <div class="c-progress" role="progressbar" aria-valuetext="Step 2 of 4" aria-valuenow="2">
+                    <div class="c-progress__step" aria-hidden="true">
+
+                        <div class="c-progress-step__item c--completed">
+                            <span class="c-progress-step__index">1</span>
+                            <div class="c-progress-step__label">Your Details</div>
+                        </div>
+
+                        <div class="c-progress-step__item c--current">
+                            <span class="c-progress-step__index">2</span>
+                            <div class="c-progress-step__label">Delivery Address</div>
+                        </div>
+
+                        <div class="c-progress-step__item">
+                            <span class="c-progress-step__index">3</span>
+                            <div class="c-progress-step__label">Secure Payment</div>
+                        </div>
+
+                        <div class="c-progress-step__item">
+                            <span class="c-progress-step__index">4</span>
+                            <div class="c-progress-step__label">Confirmation</div>
+                        </div>
+
+                    </div>
+                </div>
+
+            </div>
+        </div>
+
+        <div class="c-test__case">
+            <p class="c-test__should">It should support a varying number of steps.</p>
+            <div class="c-test__run">
+
+                <div class="c-progress u-margin-bottom-md" role="progressbar" aria-valuetext="Step 2 of 4" aria-valuenow="2">
+                    <div class="c-progress__step" aria-hidden="true">
+
+                        <div class="c-progress-step__item c--completed">
+                            <span class="c-progress-step__index">Step 1</span>
+                            <div class="c-progress-step__label">The Ingredients</div>
```

Comment:
No real reason. They could both be either or depending on the design. I updated the dust file to be spans for consistency.